CHANGE-NOTES.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+## v3.0.2
+- Enable context-menu `Override Fork Point...` action for branches that are out of sync (red-edge).
```

Comment:
if we make it we may want to change it a bit depending on the outcome of https://github.com/VirtusLab/git-machete-intellij-plugin/pull/1174

Reply:
Oh, specifically italics instead of preformatted text for action name?

Reply:
yes